client/landing/stepper/declarative-flow/internals/components/boot/index.tsx

```diff
@@ -26,20 +22,13 @@ export const Boot: FC< Props > = ( { children, fallback } ) => {
 	const locale = useLocale();
 	const newLocale = useFlowLocale();
 
-	const [ isLoadingGoalsFirst ] = useGoalsFirstExperiment();
-	const flowName = useMemo( () => getFlowFromURL(), [] );
-
 	useEffect( () => {
-		if (
-			! isReady &&
-			newLocale === locale &&
-			( flowName !== ONBOARDING_FLOW || ! isLoadingGoalsFirst )
```

Comment:
Yep, `( flowName !== ONBOARDING_FLOW || ! isLoadingGoalsFirst )` will always be truthy now.